examples/suit_update/tests/01-run.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import subprocess
+import sys
+import tempfile
+
+from testrunner import run
+
+# Default test over loopback interface
+COAP_HOST = "[fd00:dead:beef::1]"
+
+UPDATE_TIMEOUT = 120
+MANIFEST_TIMEOUT = 15
+
+USE_ETHOS = int(os.getenv("USE_ETHOS", "1"))
+TAP = os.getenv("TAP", "tap0")
+
+
+def start_aiocoap_fileserver():
+    tmpdirname = tempfile.TemporaryDirectory()
```

Comment:
I would refactor the tmpdir a bit: add it as global variable, just under `TAP`, call it `_TMP_DIR` and initialize it directly with `tempfile.TemporaryDirectory()`.
Then it can be reused from within any function. This function would just return the aiocoap_process variable. `cleanup` just takes an argument and no need to declare tmpdir `global` from several places.

What do you think ?

Reply:
> What do you think ?

done!